namespaces/cert-manager/networkpolicy.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
```

Comment:
make sure to add the namespace/pod selector for the ingress, otherwise you won't be able to reach the service

Reply:
Can't I just add the namespace selector?

Reply:
If you have explicitly labeled your namespace at creation time with `namespace: kube-system` then it should work